opencti-platform/opencti-front/src/utils/hooks/useOverviewLayoutCustomization.ts

```diff
@@ -0,0 +1,10 @@
+import useAuth from './useAuth';
+import { defaultConfiguration, OverviewLayoutCustomizationSettingsConfigurationParameters } from '../overviewLayoutCustomization';
+
+const { overviewLayoutCustomization } = useAuth();
+
+const useOverviewLayoutCustomization: (entityType: string) => Map<string, OverviewLayoutCustomizationSettingsConfigurationParameters> = (entityType) => {
```

Comment:
Should be something like :

const useOverviewLayoutCustomization: (entityType: string) => [string, number][] = (entityType) => {
  const { overviewLayoutCustomization } = useAuth();
  if (!overviewLayoutCustomization || !overviewLayoutCustomization?.has(entityType)) {
    throw Error(`Not customizable overview for ${entityType}`);
  }
  return Array.from(overviewLayoutCustomization.get(entityType)?.entries());

Reply:
I first attempted to call useAuth in the useOverviewLayoutCustomization hook but it was causing an error (illegal hook call).
We can pass the overviewLayoutCustomization to useOverviewLayoutCustomization.